### PR TITLE
feat(presets): add replacements for ZAP org moves

### PR DESCRIPTION
**Pull request by fgreinacher**
## Changes

Adds replacements for the ZAP images

## Context

The ZAP images have moved to another org twice (!) this year:

- https://www.zaproxy.org/blog/2024-03-04-zap-updates-february-2024/#owasp-docker-org-depreciation
- https://www.zaproxy.org/blog/2024-05-01-zap-updates-april-2024/#new-zaproxy-dockerhub-org

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

:hammer_and_pick: with :heart: by [Siemens](https://opensource.siemens.com/) 

/cc @nejch @dlouzan

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: lib/config/presets/internal/replacements.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:959]:
```suggestion
          'The `zap-stable` image has moved to the `zaproxy` organization.',
```

File: lib/config/presets/internal/replacements.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:971]:
```suggestion
          'The `zap-bare` image has moved to the `zaproxy` organization.',
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
We use American English, so it should be `organization`. :wink: 

See: [Renovate developer docs, Style Guide, use American English](https://github.com/renovatebot/renovate/blob/main/docs/development/style-guide.md#use-american-english).

**Review by HonkingGoose — COMMENTED** [review]
The `description` fields are good now.

I'll let a maintainer review the code.

**Review by viceice — APPROVED** [review]

**Pull request merged**
